html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <label for="Email">Email:</label>
+      <input id="Email" name="Email" type="text">
+      <div>
+        <label for="Why is coding cool?">Why is coding cool?</label>
```

Comment:
You shouldn't have spaces in a `for` or `id` value, you should use `kebab-case`, have each word separated by a dash rather than a space